src/Datadog.Trace.ClrProfiler.Managed/Integrations/ServiceStackRedisIntegration.cs

```diff
@@ -28,6 +28,7 @@ public static class ServiceStackRedisIntegration
             CallerAssembly = "ServiceStack.Redis",
             TargetAssembly = "ServiceStack.Redis",
             TargetType = "ServiceStack.Redis.RedisNativeClient",
+            TargetSignatureTypes = new[] { "T", TypeNames.Ignore, "System.Int8[][]", TypeNames.Ignore, TypeNames.Bool },
```

Comment:
```suggestion
            TargetSignatureTypes = new[] { "T", TypeNames.Ignore, "System.Byte[][]", TypeNames.Ignore, TypeNames.Bool },
```

Reply:
Wait, shouldn't the double array be the first argument after the return type "T"?